src/main/scala/uncore/tilelink2/Fuzzer.scala

```diff
@@ -34,16 +34,11 @@ class IDMapGenerator(numIds: Int) extends Module {
 
 object LFSR64
 { 
-  private var counter = 0 
-  private def next: Int = {
-    counter += 1
-    counter
-  }
-  
-  def apply(increment: Bool = Bool(true), seed: Int = next): UInt =
+  def apply(increment: Bool = Bool(true)): UInt =
   { 
     val wide = 64
-    val lfsr = RegInit(UInt((seed * 0xDEADBEEFCAFEBAB1L) >>> 1, width = wide))
+    val undef = Reg(UInt(width = wide)) // random value based on simulation seed
+    val lfsr = RegInit(Mux(undef === UInt(0), UInt(1), undef))
```

Comment:
I like this approach for testing purposes.  Have you observed how it synthesizes?  My concern is that it will degenerate to

    val lfsr = RegInit(Mux(dontCare, UInt(1), undef))

and then to 

    val lfsr = RegInit(dontCare)

and will always and forever be zero, even though the simulation model disagrees.